src/traces/box/calc.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
```

Comment:
Ahh I thought there was a check to automatically insert it already. I'll do it manually right now, but that will be a good thing to add in.